### PR TITLE
Feature/frontend review list

### PR DESCRIPTION
**Pull request by vlvereta**
Overall view of review list component made by default Bulma framework elements.
Max similar to given reference and appropriate Foursquare service element.
P.S. Added Font Awesome support into [index.blade.php](https://github.com/BinaryStudioAcademy/bsa-2018-hedonist/blob/d5bbc7caac80531e3993ed8bc8229aa3ba52efda/resources/views/index.blade.php#L10).
Also, added '/reviews' route for convenient viewing.

### CODE REVIEW

**Comment by lenchvolodymyr** [comment]
You can add yarn.lock only if you added new libraries

File: frontend/src/components/Review/ReviewList.vue [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        }
+    }
+
+</script>
+
+<style scoped>
+
+    /* Main block wrapper. */
+    .reviews-wrp {
+        width: 40%;
```
Comment (lenchvolodymyr) [inline frontend/src/components/Review/ReviewList.vue]: All looks great except width: 40%, I think parent component has to set width of child

Comment (vlvereta) [reply frontend/src/components/Review/ReviewList.vue]: I added it for nice viewing during creating/checking.
I think so too and remove that.

File: frontend/src/router/index.js [hunk 2]
```diff
@@ -1,21 +1,27 @@
-import Vue from 'vue'
-import Router from 'vue-router'
-import HelloWorld from '@/components/HelloWorld'
+import Vue from 'vue';
+import Router from 'vue-router';
+import HelloWorld from '@/components/HelloWorld';
 import ProfilePage from '@/pages/ProfilePage';
+import ReviewList from '@/components/Review/ReviewList';
```
Comment (nemoipaha) [inline frontend/src/router/index.js]: Create pages/ReviewPage component wrapper and pass all your logic there.

**Pull request merged**